### PR TITLE
utils.service: Few style-fixes

### PR DESCRIPTION
**Pull request by ldoktor**
There are few style-fixes and improved selftests related to `avocado.utils.service`. The last commit might be problematic as it changes variable-names, but I only addressed the private ones so hopefully no-one is using them.

### CODE REVIEW

**Comment by ldoktor** [comment]
@rdkdd would you please take a look at this as it's based on your PR?

File: avocado/utils/service.py [hunk 1]
```diff
@@ -661,19 +645,6 @@ class _SystemdServiceManager(_GenericServiceManager):
...
-        Create the GenericServiceManager for systemd services.
-
-        :param service_command_generator:
-        :type service_command_generator: _ServiceCommandGenerator
-        :param run: function to call to run the commands, default process.run
-        :type run: function
-        """
-        super(_SystemdServiceManager, self).__init__(service_command_generator,
-                                                     service_result_parser, run)
-
```
Comment (rdkdd) [inline avocado/utils/service.py]: yes as I understand this, parent `__init__` method is already called when parent class is initiated on line 705.

File: selftests/unit/test_utils_service.py [hunk 2]
```diff
@@ -84,22 +90,28 @@ def setUp(self):
...
+                    command_name = "chkconfig"
+                    cmd = ""
+                elif cmd == 'enable':
+                    command_name = "chkconfig"
+                    cmd = "on"
+                elif cmd == 'disable':
+                    command_name = "chkconfig"
+                    cmd = "off"
+                exp = [command_name, self.service_name, cmd]
+            self.assertEqual(ret, exp, (ret, exp, cmd))
```
Comment (rdkdd) [inline selftests/unit/test_utils_service.py]: Why did you set custom message `(ret, exp, cmd)` in `assertEqual()`?  For me implemented standard message output is better to read with hint on possible error spot:
See for yourself:
```
AssertionError: (['chkconfig', 'fake_service', 'on'], ['chkconig', 'fake_service', 'on'], 'on')
```
vs. implemented:
```
AssertionError: Lists differ: ['chkconfig', 'fake_service', ... != ['chkconig', 'fake_service', '...

First differing element 0:
chkconfig
chkconig

- ['chkconfig', 'fake_service', 'on']
?         -

+ ['chkconig', 'fake_service', 'on']

```

File: avocado/utils/service.py [hunk 3]
```diff
@@ -109,7 +109,7 @@ def sys_v_init_result_parser(command):
     if command == "status":
         def method(cmd_result):
```
[the remark this answers is not in the document]
Comment (ldoktor) [reply avocado/utils/service.py]: Oups, it shouldn't have been changed in the docstring (Eclipse did that and I didn't noticed). Sorry I'll fix that in v2.

File: avocado/utils/service.py (hunk 1 above)
Comment (ldoktor) [reply avocado/utils/service.py]: Basically this statement adds nothing. When the method is not overridden, parent's one is called. Here we override it only to call parent, so it basically does nothing.

File: selftests/unit/test_utils_service.py (hunk 2 above)
Comment (ldoktor) [reply selftests/unit/test_utils_service.py]: I changed it to see the "cmd", which is not listed on failure. Anyway I can remove that (it was useful when adding the exceptions)

**Pull request closed without merging**